packages/legend-query-builder/src/stores/QueryLoaderState.ts

```diff
@@ -212,10 +212,9 @@ export class QueryLoaderState {
     this.showingDefaultQueries = false;
     this.searchQueriesState.inProgress();
     try {
-      let searchSpecification = new QuerySearchSpecification();
-      searchSpecification.searchTerm = searchText;
+      let searchSpecification =
+        QuerySearchSpecification.createDefault(searchText);
```

Comment:
before searchSpecification.exactMatchName = true;

now with QuerySearchSpecification.createDefault(searchText);
    term.exactMatchName = false;
is this expected?

Reply:
yeah it is expected. 
we werent actuallyl sending `exactMatchName = true` to the server because the serialization for the V1 model was missing the property and hence it was always defaulting to false.